pkg/supportbundle/load.go

```diff
@@ -70,12 +74,36 @@ func ParseSupportBundleFromDoc(doc []byte) (*troubleshootv1beta2.SupportBundle,
 
 	supportBundle, ok := obj.(*troubleshootv1beta2.SupportBundle)
 	if ok {
+		// check if there is a uri field and if so,
+		// use the upstream spec, otherwise fall back to
+		// what's defined in the current spec
+		if supportBundle.Spec.Uri != "" && followURI {
+			logger.Printf("using upstream reference: %+v\n", supportBundle.Spec.Uri)
+			upstreamSupportBundleContent, err := LoadSupportBundleSpec(supportBundle.Spec.Uri)
+			if err != nil {
+				logger.Printf("failed to load upstream supportbundle, falling back")
+				return supportBundle, nil
+			}
+
+			multidocs := strings.Split(string(upstreamSupportBundleContent), "\n---\n")
+
+			upstreamSupportBundle, err := ParseSupportBundle([]byte(multidocs[0]), false)
+			if err != nil {
+				logger.Printf("failed to parse upstream supportbundle, falling back")
+				return supportBundle, nil
```

Comment:
should we be returning an error here?

Reply:
understood. i guess i'm curious what the behavior should be in the event an upstream bundle cannot be parsed. Does it fallback and return the support-bundle spec with the `URI` field to the list to be run. Do you need to always specify `collectors` and `analyzers` in your spec even when using the URI field so that it doesn't panic?

Reply:
if the upstream bundle has invalid YAML, it should fall back to simply the contents in the current bundle - but that's a good point, I didn't add a test case with _only_ a uri: field so - adding that to the to-do list